### PR TITLE
Fix memory-related tests. oops.

### PR DESCRIPTION
**Pull request by mkrutov**

### CODE REVIEW

File: test/deployment/resources/resource_request_test.go [hunk 1]
```diff
@@ -35,23 +35,23 @@ var _ = ginkgo.Describe("ResourceRequestsTests", func() {
 		deployBroker(brokerDeployer)
 		pod := getPod(ctx1)
 		actualMemRequest := pod.Spec.Containers[0].Resources.Requests.Memory()
-		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Memory limit: %s, real: %s", expectedMemLimit, actualMemLimit.String())
+		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Memory limit: %s, real: %s", expectedMemRequest, actualMemRequest.String())
```
Comment (tlbueno) [inline test/deployment/resources/resource_request_test.go:38]: I think this one was already fixed by older PR and not sure why GH still show it. 
Here is the current line from main.
https://github.com/artemiscloud/activemq-artemis-operator-test-suite/blob/d9f278bd9b4eb7a34fab44a5a582333cea13ca6c/test/deployment/resources/resource_request_test.go#L38

File: test/deployment/resources/resource_request_test.go [hunk 2]
```diff
@@ -35,23 +35,23 @@ var _ = ginkgo.Describe("ResourceRequestsTests", func() {
...
-	ginkgo.It("Memory Request check", func() {
+	ginkgo.It("Memory Request check_Update", func() {
 		expectedMemRequest := "512M"
 		brokerDeployer.WithMemRequest(expectedMemRequest)
 		deployBroker(brokerDeployer)
 		pod := getPod(ctx1)
 
 		actualMemRequest := pod.Spec.Containers[0].Resources.Requests.Memory()
-		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Memory limit: %s, real: %s", expectedMemLimit, actualMemLimit.String())
+		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Memory limit: %s, real: %s", expectedMemRequest, actualMemRequest.String())
```
Comment (tlbueno) [inline test/deployment/resources/resource_request_test.go:48]: I think this one was already fixed by older PR and not sure why GH still show it. 
Here is the current line from main.
https://github.com/artemiscloud/activemq-artemis-operator-test-suite/blob/d9f278bd9b4eb7a34fab44a5a582333cea13ca6c/test/deployment/resources/resource_request_test.go#L48

File: test/deployment/resources/resource_request_test.go [hunk 3]
```diff
@@ -35,23 +35,23 @@ var _ = ginkgo.Describe("ResourceRequestsTests", func() {
...
 
 		expectedMemRequest = "768M"
 		brokerDeployer.WithMemRequest(expectedMemRequest)
 		brokerDeployer.Update()
-		actualMemRequest := pod.Spec.Containers[0].Resources.Requests.Memory()
-		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Update to Memory limit: %s, real: %s", expectedMemLimit, actualMemLimit.String())
+		actualMemRequest = pod.Spec.Containers[0].Resources.Requests.Memory()
+		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Update to Memory limit: %s, real: %s", expectedMemRequest, actualMemRequest.String())
```
Comment (tlbueno) [inline test/deployment/resources/resource_request_test.go:54]: I think this one was already fixed by older PR and not sure why GH still show it.
Here is the current line from main.
https://github.com/artemiscloud/activemq-artemis-operator-test-suite/blob/d9f278bd9b4eb7a34fab44a5a582333cea13ca6c/test/deployment/resources/resource_request_test.go#L54

**Comment by mkrutov** [comment]
Huh. Interesting why there's no conflict according to GH.

**Comment by tlbueno** [comment]
yeah, very strange anyway.

**Pull request closed without merging**